src/back/spv/writer.rs

```diff
@@ -430,7 +430,7 @@ impl Writer {
                         constituent_ids.push(id);
 
                         if binding == &crate::Binding::BuiltIn(crate::BuiltIn::GlobalInvocationId) {
```

Comment:
I'm digging into this again for gfx-rs/wgpu#4592. Why was this one left as Global?

Reply:
It seems I missed this one 😅